src/step_block.m

```diff
@@ -0,0 +1,93 @@
+classdef step_block < matlab.System & matlab.system.mixin.Propagates
+    % step_block This block takes as input the joint torques and the
+    % applied external forces and evolves the state of the robot
+
+    % Public, tunable properties
+    properties (Nontunable)
```

Comment:
From Matlab documentation :
>By default all properties are tunable, meaning the value of the property can change at any time.
>
>Use the Nontunable attribute for a property when the algorithm depends on the value being constant once data processing starts

These properties are nontunable since they are config structures that do not have to change.
I removed the misleading comment anyway!